### DISCUSSION TITLE
v2.0.0-beta.39 requires Go 1.18+?

### DISCUSSION THREAD

**Discussion by ianmjones** [wailsapp/wails#1602] [Releases]
It looks like Wails v2.0.0-beta.39 now requires Go 1.18+ as it is using the `strings.Cut` function introduced with Go 1.18...

```shell
$ wails dev
Wails CLI v2.0.0-beta.39

Executing: go mod tidy -compat=1.17
Executing: /nix/store/0d6pk4m6x815z16b42543rz93fr6ijry-wails-2.0.0-beta.39/bin/wails generate module


ERROR:
# github.com/wailsapp/wails/v2/internal/binding
/home/ian/go/pkg/mod/github.com/wailsapp/wails/v2@v2.0.0-beta.39/internal/binding/generate.go:132:28: undefined: strings.Cut

exit status 2

If Wails is useful to you or your company, please consider sponsoring the project:
https://github.com/sponsors/leaanthony


exit status 1
```

Was this an intentional change?

If so, @leaanthony are you happy for me to submit a PR that updates all references in the `beta.39` and `current` versions of the website and v2 source and templates code to use Go 1.18? (not sure whether there's some system in place already to update all the go.mod templates to ensure their references are tidy)

If not intentional, I guess I've found a bug! :smile:

**Comment by leaanthony** [comment]
It was not intentional 😔 I'll need to patch.

**Reply by leaanthony** [reply]
@ianmjones https://github.com/wailsapp/wails/pull/1605

**Reply by ianmjones** [reply]
Just tested v2.0.0-beta.39.2, works fine for me.
